src/Illuminate/Collections/Enumerable.php

```diff
@@ -1000,6 +1000,14 @@ public function zip($items);
      */
     public function collect();
 
+    /**
+     * Get a single property from the first element in the collection.
+     *
+     * @param  string  $key
+     * @return mixed
+     */
+    public function value($key);
```

Comment:
This is a breaking change unfortunately so please send this to master.

Reply:
I'll open a new PR.
May I ask how this breaks existing functionality?

Reply:
Adding a new method to an interface will break all third party implementations of that interface because they aren't implementing the method yet.

Reply:
To be clear for the future: we can definitely add additional methods to the collections and `EnumeratesValues` mid-cycle.

They should just not be added to the interface until the next release.